subsys/bluetooth/controller/ll_sw/ull_conn_types.h

```diff
@@ -158,6 +158,15 @@ struct ll_conn {
 	struct ull_hdr  ull;
 	struct lll_conn lll;
 
+#if defined(CONFIG_BT_CTLR_SYNC_TRANSFER_RECEIVER)
+	struct {
+		uint8_t mode;
+		uint8_t cte_type;
+		uint16_t skip;
+		uint16_t timeout;
+	} past;
```

Comment:
Would it make sense to define this struct by itself and use that in `ull_conn`, instead of the 4 individual values? Would make it easier to reset, and could even return those values as a reference or a single value instead of having 4 functions so that

```c
	conn->past.mode    = ull_conn_default_past_param_mode_get();
	conn->past.skip    = ull_conn_default_past_param_skip_get();
	conn->past.timeout = ull_conn_default_past_param_timeout_get();
	conn->past.cte_type = ull_conn_default_past_param_ctetype_get();
```

could become something like 

```c
	conn->past = ull_conn_default_past_param_get();
```
or

```c
	memcpy(&conn->past, ull_conn_default_past_param_get(), sizeof(conn->past));
```

or is there some value in having 4 separate functions and values for this?

Reply:
No, I think it will make perfect sense to make it into a single get function instead of four.

I'll make a change for that :)